src/index.ts

```diff
@@ -303,6 +303,46 @@ export const jupyterCommentingPlugin: JupyterFrontEndPlugin<ICommentPanel> = {
 
     // Add the panel to the shell's right area.
     shell.add(panel, 'right', { rank: 600 });
+    const button = panel.button;
+
+    function openButton(x: number, y: number, anchor: HTMLElement): void {
+      button.open(
+        x,
+        y,
+        () => {
+          let editorWidget = (shell.currentWidget as DocumentWidget)
+            .content as CodeEditorWrapper;
+
+          if (editorWidget == null) {
+            return;
+          }
+
+          const model = panel.model;
+          if (model == null) {
+            return;
+          }
+
+          const comments = model.comments;
```

Comment:
I think this is unnecessary--it's for placing the comment at the right spot in the notebook files but doesn't do anything here